--- samples.paths/xmlcomments	Tue Dec  1 03:56:34 1998
+++ samples/xmlcomments	Wed Oct 20 15:10:47 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # $Revision$
 #
--- samples.paths/xmlfilter	Sun Mar 21 23:46:16 1999
+++ samples/xmlfilter	Wed Oct 20 15:10:53 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # $Revision$
 #
--- samples.paths/xmlstats	Wed Oct 28 05:30:23 1998
+++ samples/xmlstats	Wed Oct 20 15:10:57 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # $Revision$
 #
